pkg/storage/replica_proposal.go

```diff
@@ -470,6 +482,39 @@ func (r *Replica) leasePostApply(
 	}
 }
 
+func (r *Replica) addSSTablePostApply(ctx context.Context, data []byte) {
+	var checksum []byte
+	{
+		hash := crc32.New(crc32.MakeTable(crc32.Castagnoli))
```

Comment:
I somehow imagined we'd compute the checksum along with the SSTable and send with the request. That way, we catch more corruption and besides, it's more opportune to do it where we create the SSTable in the first place, no?